### PR TITLE
OPSEXP-847: Change base URL's for Nexus

### PR DESCRIPTION
**Pull request by nilsSure**

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -39,38 +39,38 @@ sync:
...
+  acs_zip_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}/alfresco-content-services-distribution/{{ acs.version }}/alfresco-content-services-distribution-{{ acs.version }}.zip.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
+  adw_zip_url: "{{ nexus_repository.enterprise_releases }}/alfresco-digital-workspace/{{ adw.version }}/alfresco-digital-workspace-{{ adw.version }}.zip"
+  adw_zip_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}/alfresco-digital-workspace/{{ adw.version }}/alfresco-digital-workspace-{{ adw.version }}.zip.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
+  search_zip_url: "{{ nexus_repository.releases }}/alfresco-search-services/{{ search.version }}/alfresco-search-services-{{ search.version }}.zip"
+  search_zip_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.releases }}/alfresco-search-services/{{ search.version }}/alfresco-search-services-{{ search.version }}.zip.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
```
Comment (gavincornwell) [inline group_vars/all.yml:47]: Not sure if we missed this one before or it regressed but this one shouldn't need nexus credentials as it's public

File: group_vars/all.yml [hunk 2]
```diff
@@ -39,38 +39,38 @@ sync:
...
+  trouter_jar_url: "{{ nexus_repository.enterprise_releases }}/alfresco-transform-router/{{ trouter.version }}/alfresco-transform-router-{{ trouter.version }}.jar"
+  trouter_jar_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}/alfresco-transform-router/{{ trouter.version }}/alfresco-transform-router-{{ trouter.version }}.jar.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
+  transform_jar_url: "{{ nexus_repository.releases }}/alfresco-transform-core-aio-boot/{{ transform.version }}/alfresco-transform-core-aio-boot-{{ transform.version }}.jar"
+  transform_jar_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.releases }}/alfresco-transform-core-aio-boot/{{ transform.version }}/alfresco-transform-core-aio-boot-{{ transform.version }}.jar.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
```
Comment (gavincornwell) [inline group_vars/all.yml:53]: Not sure if we missed this one before or it regressed but this one shouldn't need nexus credentials as it's public

**Review by gavincornwell — CHANGES_REQUESTED** [review]
Other than a couple more URLs with nexus creds that don't need them, it looks good to me.

File: group_vars/all.yml (hunk 1 above)
Comment (nilsSure) [reply group_vars/all.yml:47]: will attend to them

**Review by gavincornwell — APPROVED** [review]
LGTM, approved, pending a green build

**Pull request merged**